iocBoot/iocIpmiMgr/stIpmiMgr.cmd

```diff
@@ -13,9 +13,8 @@ drvAsynIPPortConfigure ("$(PORT)","$(IPADDR):623 udp",0,0,0)
 mchInit("$(PORT)")
 
 ## Load record instances
-dbLoadRecords("$(ASYN)/db/asynRecord.db","P=${P}, R=${R}asyn,PORT=$(PORT),ADDR=0,OMAX=80,IMAX=80")
-dbLoadRecords("${TOP}/db/microtca_bpm_crate.db", "link=$(PORT),P=$(P),R=$(R)")
-
+dbLoadRecords("$(TOP)/db/asynRecord.db","P=${P}, R=${R}asyn,PORT=$(PORT),ADDR=0,OMAX=80,IMAX=80")
+dbLoadRecords("${TOP}/db/microtca_bpm_crate.db", "link=$(PORT),P=$(P),R=$(R),dev=$(P)$(R),unit='',aliasprefix=''")
```

Comment:
Maybe we can have `P` (or `R`) defined without the final `:` so that the `dev` does not contain duplicated `::`.